<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsAPP1Class</class>
 <widget class="QMainWindow" name="QtWidgetsAPP1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1508</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsAPP1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>761</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="fileButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>导入图片集合</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="filePath">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame_img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>631</width>
       <height>591</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QListWidget" name="imgListWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>631</width>
        <height>581</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="paintFixed">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>固定像素绘制</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paintGPU">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>GPU加速绘制</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paintMarching">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>90</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>面绘制</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_ImageShow">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>561</width>
      <height>761</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QVTKWidget" name="qvtkWidget_later">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>521</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QVTKWidget" name="qvtkWidget_before">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>521</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_ITKFIilter">
    <property name="geometry">
     <rect>
      <x>1249</x>
      <y>29</y>
      <width>221</width>
      <height>721</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="filterButton_median">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>中值滤波</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileButton_Erode">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>形态学腐蚀</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileButton_Dilate">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>形态学膨胀</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileButton_Shift">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>线性阈值变换</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileButton_Canny">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Canny边缘检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileButton_Gaussian">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>370</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>边缘检测(Gaussian)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileButton_Discrete">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>440</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>离散高斯模糊</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileButton_Threshold">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>510</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>阈值过滤</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileButton_Sigmoid">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>580</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>非线性滤波</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileButton_Normal">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>650</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>像素归一化</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1508</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtWidgetsAPP1.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>clicked_on_fileBtn()</slot>
 </slots>
</ui>
